templates/contact-us/email.md

```diff
@@ -6,8 +6,9 @@ Hello,
 
 This person is trying to reach you:
 
-* Email: {{reply_to}}
-* Message: {{message}}
+{{#info}}
+  * {{key}}: {{value}}
+{{/info}}
 
 Truly yours,  
 Sent by [Enspirit.be](https://enspirit.be/), contact us if you need help with any IT task.
```

Comment:
Can you try whether it would be possible to tune the email Subject and this footer phrase from environment variables?